src/lib/output/renderer.ts

```diff
@@ -201,7 +201,24 @@ export class Renderer extends ChildableComponent<Application, RendererComponent>
                 }
             }
 
-            this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            let filename = Path.join(path, 'theme.js');
+            if (!FS.existsSync(filename)) {
+                this.theme = this.addComponent('theme', new DefaultTheme(this, path));
+            } else {
+                try {
+                    const plugin = require(filename);
+                    if(typeof plugin === 'function') {
+                        plugin(this.application);
```

Comment:
I don't follow what you mean... I'm not terribly familiar with module exports given that there are umd, commonjs, amd and a slew of others.  

Can you give me an example of what you mean?  I thought I implemented this the way you requested.